game/models.py

```diff
@@ -585,13 +586,44 @@ class Meta:
         )
 
 
-class CompetitionManager(models.Manager):
+class CompetitionMixin(object):
```

Comment:
Ok that's another solution. I did the chaining on purpose .. like I'd have done in rails. I find it pretty handy, but let's discuss that.